src/point_one/fusion_engine/messages/solution.h

```diff
@@ -42,28 +42,40 @@ struct PoseMessage {
   double lla_deg[3] = {NAN, NAN, NAN};
 
   /**
-   * The platform attitude (in degrees), if known, described as Euler-321 angles
-   * (yaw, pitch, roll), or `NAN` if attitude has not been initialized.
+   * The position standard deviation (in meters), resolved in the local level
```

Comment:
I think most people refer to this as in the Local Tangent Plane

Reply:
Yeah I've seen both but that seems more popular. Will change.